Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/rtl/HindiNumberFormatTest.java

```diff
@@ -0,0 +1,93 @@
+package org.catrobat.paintroid.test.espresso.rtl;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.catrobat.paintroid.MainActivity;
+import org.catrobat.paintroid.R;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Locale;
+
+import static android.support.test.espresso.Espresso.onData;
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.catrobat.paintroid.test.espresso.util.EspressoUtils.clickColorPickerPresetSelectorButton;
+import static org.catrobat.paintroid.test.espresso.util.EspressoUtils.openColorPickerDialog;
+import static org.catrobat.paintroid.test.espresso.util.EspressoUtils.openNavigationDrawer;
+import static org.catrobat.paintroid.test.espresso.util.UiMatcher.withBackground;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.hasToString;
+import static org.hamcrest.core.StringStartsWith.startsWith;
+
+@RunWith(AndroidJUnit4.class)
+public class HindiNumberFormatTest {
+    private static final Locale ARABICLOCALE = new Locale("ar");
+    private static final String EXPECTED_RED_VALUE = "٢٤٠";
+    private static final String EXPECTED_GREEN_VALUE = "٢٢٨";
+    private static final String EXPECTED_BLAU_VALUE = "١٦٨";
+    private static final String EXPECTED_ALFA_VALUE = "١٠٠";
+    private static final String EXPECTED_STROKE_WIDTH_VALUE = "٢٥";
+    private static final String EXPECTED_COLOR_TOLERANCE_VALUE = "١٢";
+
+    @Rule
+    public ActivityTestRule<MainActivity> launchActivityRule = new ActivityTestRule<>(MainActivity.class);
+
+    @Test
+    public void testHindiNumberAtBrushTool() throws Exception {
+        openNavigationDrawer();
+        onView(withText(R.string.menu_language))
+                .perform(click());
+        onData(hasToString(startsWith(ARABICLOCALE.getDisplayName(ARABICLOCALE))))
+                .perform(click());
+
+        onView(withId(R.id.tools_brush))
```

Comment:
please have a look at our 
[ExpressoUtils](https://github.com/Catrobat/Paintroid/blob/develop/Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/util/EspressoUtils.java)
Please use these helper functions. If they are not working for RTL, write new ones for RTL
Example: instead of onView(with(Id(R.id.tools_brush)) you can use selectTool(ToolType.BRUSH)